src/auth/email/mod.rs

```diff
@@ -0,0 +1,72 @@
+use lettre::message::header::ContentType;
+use lettre::message::Mailbox;
+use lettre::transport::smtp::authentication::Credentials;
+use lettre::{Address, Message, SmtpTransport, Transport};
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum MailError {
+    #[error("Environment Error")]
+    Env(#[from] dotenvy::Error),
+    #[error("Mail Error: {0}")]
+    Mail(#[from] lettre::error::Error),
+    #[error("Address Parse Error: {0}")]
+    Address(#[from] lettre::address::AddressError),
+    #[error("SMTP Error: {0}")]
+    Smtp(#[from] lettre::transport::smtp::Error),
+}
+
+pub fn send_email_raw(to: String, subject: String, body: String) -> Result<(), MailError> {
```

Comment:
Lots of unwraps here in this function. I don't think that any of are guaranteed to be Ok()